vendor/engines/secure_rooms/app/services/secure_rooms/access_rules/operator_rule.rb

```diff
@@ -5,7 +5,7 @@ module AccessRules
     class OperatorRule < BaseRule
 
       def evaluate
-        grant! if user.operator_of?(card_reader.facility)
+        grant! if user.operator_of?(card_reader.facility) && !user.administrator?
```

Comment:
I don't think this is quite the right fix. There's the case where a user is an operator_of _and_ is a global admin. I think in that case, we would want to grant.